docs/source/serving/spot-policy.rst

```diff
@@ -0,0 +1,140 @@
+.. _spot_policy:
+
+Using Spot Instances
+====================
+
+SkyServe supports serving models on a mixture of spot and on-demand replicas with two options: :code:`base_ondemand_fallback_replicas` and :code:`dynamic_ondemand_fallback`.
+
+:code:`base_ondemand_fallback_replicas` sets the number of on-demand replicas to keep running at all times. This is useful for ensuring service availability and making sure that there is always some capacity available, even if spot replicas are not available. :code:`use_spot` should be set to :code:`true` to enable spot replicas.
+
+.. code-block:: yaml
+
+    service:
+        readiness_probe: /health
+        replica_policy:
+            min_replicas: 2
+            max_replicas: 3
+            target_qps_per_replica: 1
+            base_ondemand_fallback_replicas: 1
+
+    resources:
+        ports: 8081
+        cpus: 2+
+        use_spot: true
+
+    workdir: examples/serve/http_server
+
+    run: python3 server.py
+
+Dynamic On-demand Fallback
+==========================
+
+SkyServe supports dynamically fallback to on-demand replicas when spot replicas are not available. This is enabled by setting :code:`dynamic_ondemand_fallback` to be :code:`true`. This is useful for ensuring the required capacity of replicas in the case of spot instance interruptions. When spot replicas are available, SkyServe will automatically switch back to using spot replicas to maximize cost savings. SkyServe supports both specifying :code:`base_ondemand_fallback_replicas` and :code:`dynamic_ondemand_fallback`.
```

Comment:
Maybe we should put `SkyServe supports both specifying :code:base_ondemand_fallback_replicas and :code:dynamic_ondemand_fallback.` in a tip/hint box for visibility?